### PR TITLE
Update the changelog for v0.9.16

### PR DESCRIPTION
**Pull request by Amazinite**
You know the drill.

### CODE REVIEW

File: changelog [hunk 1]
```diff
@@ -1,3 +1,54 @@
...
+      * Changed a Remnant bounty mission that had an unconstrained npc with a jump drive to spawn one without a jump drive as to stop the ship from escaping the Ember Waste. (@Hecter94)
+    * Engine bugs:
+      * The load panel now only lists .txt files from the save folder instead of listing every file there. (@quyykk)
+      * Traveling between systems no longer assumes that hyperdrives are always cheaper to use than jump drives. (@RisingLeaf, @Amazinite)
+      * Fixed an error where attempting to launch with a fighter as your flagship under certain circumstances could cause you to launch with no ship at all. (@petervdmeer)
+      * Mission npcs that become disabled by corrosion damage will now properly have the disable objective set. (@RisingLeaf)
+      * Disallowed planet labels from jumping around at different zoom levels for a better viewing experience. (@quyykk)
+      * Fixed a longstanding bug where npcs could get stuck thinking they need fuel after traveling through a wormhole. (@yjhn, @warp-core)
```
Comment (warp-core) [inline changelog:30]:
```suggestion
      * Fixed a longstanding bug where npcs could get stuck thinking they need fuel after traveling through a wormhole. (@yjhn)
```
I didn't really do much here.

File: changelog [hunk 2]
```diff
@@ -1,3 +1,54 @@
+Version 0.9.16:
+  * Breaking changes:
+    * The accompany objective on mission npcs no longer implicitly sets the save objective. Plugin missions that had "accompany" but not "save" should be changed to include "save" if the intention of the mission is that the npcs live. (@RisingLeaf)
+  * Bug fixes:
+    * Content bugs:
+      * Typo fixes. (@a-random-lemurian, @arachi-lover, @LepRyot, @tibetiroka)
+      * The Firestorm Battery outfit now has a proper plural name. (@warp-core)
+      * The Mule's description has been updated to reflect its new category. (@McloughlinGuy)
+      * Corrected the name of the passenger in "FW Pug 4." (@a-random-lemurian, @williaji)
+      * Added a firing sound to the ka'het Annihilator Turret, as it was missing one. (@Hecter94)
```
Comment (tibetiroka) [inline changelog:10]:
```suggestion
      * Added a firing sound to the Ka'het Annihilator Turret, as it was missing one. (@Hecter94)
```

File: changelog [hunk 3]
```diff
@@ -1,3 +1,54 @@
...
+      * Fixed a longstanding bug where npcs could get stuck thinking they need fuel after traveling through a wormhole. (@yjhn, @warp-core)
+      * Fixed a bug where fighters would sometimes fail to reparent to a carrier that had space for them when recalled. (@samrocketman)
+  * Game content:
+    * Balance:
+      * Added H2H weapons to some Hai ships and increased the base crew stats of the Unfettered. (@Hurleveur)
+    * Graphics:
+      * Slightly reworked the atomic and ion flare effects. (@GefullteTaubenbrust2)
+    * Other:
+      * Recategorized the Hai Centipede as a space liner. (@1010todd)
+      * Changed the landscaope images of Mainsail and Inmost Blue to better fit their planet descriptions. (@Anarchist2)
```
Comment (tibetiroka) [inline changelog:39]:
```suggestion
      * Changed the landscape images of Mainsail and Inmost Blue to better fit their planet descriptions. (@Anarchist2)
```

**Review by tibetiroka — CHANGES_REQUESTED** [review]

**Review by roadrunner56 — APPROVED** [review]

**Review by Hecter94 — APPROVED** [review]

**Comment by Saugia** [comment]
Do I? Do I know the drill? I know a drill. But I don't know *the* drill.

**Review by Saugia — APPROVED** [review]

**Pull request merged**